platform/io/platform-io-impl/src/main/java/org/codice/ddf/platform/io/FelixConfigPersistenceStrategy.java

```diff
@@ -0,0 +1,48 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ **/
+package org.codice.ddf.platform.io;
+
+import static org.apache.commons.lang.Validate.notNull;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.util.Dictionary;
+
+import org.apache.commons.lang.Validate;
+import org.apache.felix.cm.file.ConfigurationHandler;
+import org.codice.ddf.platform.io.internal.PersistenceStrategy;
+
+/**
+ * Class that persists configuration properties using the Felix config file format.
+ */
+public class FelixConfigPersistenceStrategy implements PersistenceStrategy {
```

Comment:
The naming for this class (`FelixConfigPersistenceStrategy `) and `FelixCfgPersistenceStrategy` seems confusing to me, is there any better way to distinguish these classes from each other? Maybe add Format in the name? ✏️